### ISSUE TITLE
Support for manual schema?

### ISSUE THREAD

**Issue by Crash0v3rrid3** [tfranzel/drf-spectacular#998]
In the drf package, the `rest_framework.auth_token.views.ObtainAuthToken` class defines & assigns a `rest_framework.schemas.coreapi.ManualSchema` object to the schema attribute. The ManualSchema class doesn't inherit the AutoSchema and hence I get the below exception. 
```
AssertionError: Incompatible AutoSchema used on View <class 'rest_framework.authtoken.views.ObtainAuthToken'>. 
Is DRF's DEFAULT_SCHEMA_CLASS pointing to "drf_spectacular.openapi.AutoSchema" or any other drf-spectacular compatible AutoSchema?
```
I see that the SchemaGenerator class has an explicit assert to check if the schema definition is an object of drf_spectacular.openapi.AutoSchema. Is it because drf-spectacular doesn't support ManualSchema? Is there a way to handle this through drf-spectacular?

I'm currently using,
Django==2.2.16
djangorestframework==3.10.0
drf-spectacular==0.21.1

**Comment by tfranzel** [comment]
> requirements: Django REST Framework (3.10.3, 3.11, 3.12, 3.13, 3.14)

your DRF version (3.10.0) is not supported. update to at least 3.10.3

> drf-spectacular==0.21.1

This error is a bug in DRF that we accounted for in #803. This is fixed starting with 0.24.0. update your spectacular version to most recent.

**Comment by Crash0v3rrid3** [comment]
After upgrading spectacular to 3.24.2, the issue was resolved.

Closing the issue.
